### PR TITLE
Eth2 updates and InfoBanner update [#1928, #1927]

### PR DESCRIPTION
**Pull request by ryancreatescopy**
## Description

- Added new Eth2 articles
- Removed mention of beacon chain needing x deposits and replaced it with copy saying that we've now got enough deposits to launch
- Refactored InfoBanner which was displaying weirdly on mobile – aligned it with Warning design. Had to tweak a few instances of it to make sure it displayed correctly.
- Fixed dodgy alignment of cards on beacon chain page.
- While working on this PR I discovered I'd used the wrong string for the dapps page title in the nav. Have fixed that too... this caused the selectable tag on the find-wallet page to read incorrectly.

## Related Issues

#1928 
#1927

### CODE REVIEW

File: src/pages-conditional/wallets/find-wallet.js [hunk 1]
```diff
@@ -87,6 +86,14 @@ const Header = styled.header`
   }
 `
 
+const InfoBannerContainer = styled.div`
```
Comment (samajammin) [inline src/pages-conditional/wallets/find-wallet.js:89]: Why not just style `<InfoBanner/>` directly vs. creating this container?

File: src/pages-conditional/wallets/find-wallet.js [hunk 2]
```diff
@@ -87,6 +86,14 @@ const Header = styled.header`
...
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    margin-left: 2rem;
+    margin-right: 2rem;
+    margin-top: -2rem;
```
Comment (samajammin) [inline src/pages-conditional/wallets/find-wallet.js:93]: Why negative margin? I suspect there's a better way to solve this, e.g. by reducing margin-top of the element above.

File: src/pages/get-eth.js [hunk 3]
```diff
@@ -331,14 +331,15 @@ const GetETHPage = ({ data }) => {
               <Translation id="page-get-eth-." />
             </em>
           </p>
+          <br />
```
Comment (samajammin) [inline src/pages/get-eth.js:334]: Perhaps we add margin-top vs. a `<br/>`? They can act a bit unpredictably.

**Review by samajammin — CHANGES_REQUESTED** [review]
Looks good for the most part but a couple questions/comments. Happy to pair on this tomorrow.

File: src/pages-conditional/wallets/find-wallet.js (hunk 1 above)
Comment (ryancreatescopy) [reply src/pages-conditional/wallets/find-wallet.js:89]: It makes the banner go full width as it's not in a parent container with any padding/margin set.

File: src/pages-conditional/wallets/find-wallet.js (hunk 2 above)
Comment (ryancreatescopy) [reply src/pages-conditional/wallets/find-wallet.js:93]: Changed margins of the container above.

File: src/pages/get-eth.js (hunk 3 above)
Comment (ryancreatescopy) [reply src/pages/get-eth.js]: done.

**Review by samajammin — APPROVED** [review]

**Pull request merged**